framework/src/Volo.Abp.AspNetCore.Components.WebAssembly/Volo/Abp/AspNetCore/Components/WebAssembly/AbpAspNetCoreComponentsWebAssemblyModule.cs

```diff
@@ -18,7 +19,8 @@ namespace Volo.Abp.AspNetCore.Components.WebAssembly;
 [DependsOn(
     typeof(AbpAspNetCoreMvcClientCommonModule),
     typeof(AbpUiModule),
-    typeof(AbpAspNetCoreComponentsWebModule)
+    typeof(AbpAspNetCoreComponentsWebModule),
+    typeof(AbpEventBusModule)
```

Comment:
Why adding this to dependencies? Volo.Abp.AspNetCore.Components.WebAssembly.csproj has no such a project reference.

Reply:
I add a `ClientProxyExceptionEventHandler` in this project.

Reply:
Will add the project reference too.

Reply:
If there is no project reference, but you can use the module (that defines ClientProxyExceptionEventHandler), it means you indirectly reference the project. In this case, you should have an indirect module dependency. So, you should not define the module dependency, it should also come indirectly.